base/echo/base/deployment.yml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echo
+  labels:
+    app.kubernetes.io/name: echo
+    app.kubernetes.io/instance: echo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: echo
+      app.kubernetes.io/instance: echo
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: echo
+        app.kubernetes.io/instance: echo
+    spec:
+      containers:
+      - env:
+        - name: JAVA_OPTS
+          value: -XX:MaxRAMPercentage=80.0
+        - name: SPRING_PROFILES_ACTIVE
+          value: local
+        image: gcr.io/spinnaker-marketplace/echo:spinnaker-master-latest-unvalidated
+        name: echo
+        ports:
+        - name: traffic-port
+          containerPort: 8089
+          protocol: TCP
+        readinessProbe:
+          httpGet:
+            port: traffic-port
+            path: /health
+          failureThreshold: 3
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        resources:
```

Comment:
I also blindly copied this block from Clouddriver; even though it's empty on Halyard-deployed Echo. Were these values meant to be Clouddriver-specific or reasonable for all services?

Reply:
These will probably need some tuning---they're probably the most common things people will configure themselves, but we should still try to have reasonable defaults.  My guess is maybe we could go a bit lower for `echo`?  But maybe what we should do after everything deploys is make a reasonable size deployment and see how good our estimates are, and tune it then...at least as best as we can for defaults.

Reply:
Okay that sounds great; I'll leave these for now but I added a todo in our kleat doc to edit these based on tuning when we've completed all the services.